wordpress-mobile/WordPress-Android
# Issue/10433 semantic color names in xml layouts

Part of #10433

This PR is intended to be a part of a gradual effort to get our styling in order and reduce tech debt.

In this PR I changed direct color references to their semantic attributes in XML layouts. I did it only in layouts where it makes sense and the change is straightforward. I left out some layouts that will require extra TLC and will address them in subsequent PR's.

This PR does not carry any visual changes. All the changes are semantic.

I identified the following color groups based on common usage

| Semantic Attribute Name | Semantic color name (color studio name)   | iOS name   |
|-------------------------|----------------------|------------|
| wpColorTextTetriary             | neutral_40 (gray_40) | textTetriary       |
| wpColorTextQuaternary       | neutral_30 (gray_30)    | textQuaternary |
| wpColorTextInverted       |  @ android:color/white    | textInverted  |
| wpColorAccent       | accent_50 (pink_50)    | accent |
| wpColorAccentLight       | accent_30 (pink_30)    | N/A |
| wpColorError       | error_50 (red_50)    | error |
| wpColorPrimary       | primary (wordpress_blue_50)    | primary |
| wpColorPrimaryLight       | primary_40 (wordpress_blue_40)    | primaryLight |
| wpColorPrimaryDark       | primary_70 (wordpress_blue_70)    | primaryDark |
| wpColorSuccess       | success_50 (green_50)    | success |
| wpColorWarningDark       | warning_50 (yellow_50)    | warning |
| wpColorWarningLight       | warning (yellow_20)    | N/A |
| wpColorBasicBackground       | neutral_0 (gray_0)    | basicBackground |
| wpColorSurface       | @ android:color/white    | listForeground (?) |
| wpColorDivider       | gray_5 | divider |
| wpColorAppBar       | primary_50 (wordpress_blue_50)    | appBar |

PS: I also fixed usage of white color, by switching from `@color/white` to `@android:color/white`

Colors that do not fit into this table are used sparingly and will require some additional attention to determine if we going to change them to more commonly used semantic variations, add them to the list or leave them be.


To test:
- From developer - app builds, and changes in XML files are limited to switching from color references to attributes.
- From designer - there is no visual change, I would just love to have a second pair of eyes to make sure everything looks the same. Maybe any comments on semantic color names?

Update release notes:
